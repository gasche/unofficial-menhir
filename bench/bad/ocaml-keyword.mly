%type <int * int> constraint
%start constraint
%start type
%token EQ

%%

constraint:
  type EQ type
    { $1, $3 }

type:
  INT
    { TypInt }

