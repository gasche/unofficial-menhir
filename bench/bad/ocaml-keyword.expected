File "ocaml-keyword.mly", line 1, characters 18-28:
Error: this is an Objective Caml reserved word.
